scripts/os_configure/install_software_20.bash

```diff
@@ -62,23 +94,19 @@ bash $SCRIPT_PATH/install_software_ros2.bash $ROS2_VERSION
 sudo rosdep init
 rosdep update
 
-# if ([[ $computer_type == "lab" ]] || [[$computer_type == "robot" ]])
-# then
-#   bash $SCRIPT_PATH/configure_ROSTeamWS.bash $ROS_VERSION
-# fi
-#
-# if ([[ $computer_type == "robot" ]])
-# then
-#   bash $SCRIPT_PATH/configure_robots.bash $ROS_VERSION
-# fi
-
-if ([[ $computer_type == "office" ]])
+if [[ $computer_type != "robot" ]]
 then
+  sudo apt -y install recordmydesktop rdesktop gimp meshlab inkscape pdfposter unrar
+  sudo DEBIAN_FRONTEND=noninteractive apt -y install wireshark
 
-  sudo apt -y install pass
+  # Latex
+  sudo apt -y install kile texlive-full texlive-lang-german kbibtex ktikz
```

Comment:
This should be only used with office computers.

Reply:
Done